From 416c3d28199d5f3db185c6a97607f4c0df856e84 Mon Sep 17 00:00:00 2001
From: Andoni Morales Alastruey <ylatuya@gmail.com>
Date: Tue, 17 Mar 2015 20:42:30 +0100
Subject: [PATCH] Fix restore command

---
 .nuget/NuGet.targets | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.nuget/NuGet.targets b/.nuget/NuGet.targets
index 46a1b6c..d090a78 100644
--- a/.nuget/NuGet.targets
+++ b/.nuget/NuGet.targets
@@ -51,7 +51,7 @@
         <NonInteractiveSwitch Condition=" '$(VisualStudioVersion)' != '' AND '$(OS)' == 'Windows_NT' ">-NonInteractive</NonInteractiveSwitch>
         
         <!-- Commands -->
-        <RestoreCommand>$(NuGetCommand) install "$(PackagesConfig)" -source "$(PackageSources)"  $(NonInteractiveSwitch) $(RequireConsentSwitch) -solutionDir "$(SolutionDir) " </RestoreCommand>
+        <RestoreCommand>$(NuGetCommand) install "$(PackagesConfig)" -source "$(PackageSources)"  $(NonInteractiveSwitch) $(RequireConsentSwitch) -solutionDir "$(SolutionDir)" </RestoreCommand>
         <BuildCommand>$(NuGetCommand) pack "$(ProjectPath)" -Properties Configuration=$(Configuration) $(NonInteractiveSwitch) -OutputDirectory "$(PackageOutputDir)" -symbols</BuildCommand>
 
         <!-- We need to ensure packages are restored prior to assembly resolve -->
-- 
1.9.1

